logviewer/src/main/java/com/nodesagency/logviewer/data/database/daos/LogEntryDao.kt

```diff
@@ -18,4 +18,13 @@ internal interface LogEntryDao {
     @Insert
     fun insert(entry: LogEntry)
 
```

Comment:
Similar question regarding lowercasing for `getLogsWithSeverity`, `getLogsWithMessage` and `getLogsWithTag`

> Will this return the proper category if the :name is "I/O", but the name (the stored value compared) is lowercased to "i/o"?

Reply:
LIKE is supposed to be case insensitive so I've removed lower() completely. Will cover this case with unit tests later